packages/push/src/PushRegistration.ts

```diff
@@ -21,7 +29,13 @@ export class PushRegistration {
   public static readonly TYPE: string = "push";
   public static readonly REGISTRATION_DATA_KEY = "UPS_REGISTRATION_DATA";
 
+  public static onMessageReceived(onMessageReceivedCallback: OnMessageReceivedCallback) {
+    PushRegistration.onMessageReceivedCallback = onMessageReceivedCallback;
+  }
+
+  private static readonly REGISTRATION_TIMEOUT = 5000;
```

Comment:
TBH this might be too low considering that push had timeout issues before with limit that was 2 minutes